code/_globalvars/lists/objects.dm

```diff
@@ -48,8 +47,6 @@ GLOBAL_LIST_EMPTY(ladders)
 GLOBAL_LIST_INIT(active_diseases, list()) 			//List of Active disease in all mobs; purely for quick referencing.
 
 GLOBAL_LIST_EMPTY(mob_spawners) 		    // All mob_spawn objects
-GLOBAL_LIST_EMPTY(alert_consoles)			// Station alert consoles, /obj/machinery/computer/station_alert
```

Comment:
the is one of those things that should remain a global for consistency and future proof purposes.

Reply:
I definitely agree that this could be useful for in the future for something, but currently there's no benefit at all in it being there. I can't think of any admin situations where the list would be useful either, and in the event it *was* needed, the same effect can be achieved with far less hassle using an SDQL2 query.

I could change it to just be commented out instead to avoid someone needing to remake it again in the future, but personally at least I can't see much reason for this to stay.